api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1524,12 +1524,14 @@ def trash_container(self) -> Union[labware.Labware, TrashBin, WasteChute]:
         This is the property used to determine where to drop tips and blow out liquids
         when calling :py:meth:`drop_tip` or :py:meth:`blow_out` without arguments.
 
-        On a Flex running a protocol with API version 2.16 or higher, ``trash_container`` is
-        the first ``TrashBin`` or ``WasteChute`` object loaded in the protocol.
-        On a Flex running a protocol with API version 2.15, ``trash_container`` is
-        a single-well fixed trash labware in slot D3.
-        On a an OT-2, ``trash_container`` is always a single-well fixed trash labware
-        in slot 12.
+        The default value depends on the robot type and API version:
```

Comment:
Up to you, but we may want to explicitly say that this is overridable. I don't think Sphinx renders any distinction between read-only properties and writable ones.

```suggestion
        You can set this to a :py:obj:`Labware`, :py:obj:`TrashBin`, or :py:obj:`WasteChute`.

        The default value depends on the robot type and API version:
```

Note that I'm cross-referencing to `WasteChute` and `TrashBin` here, which aren't currently published, so the links won't go anywhere. I think we should publish them, though; they're like `Labware`.

Reply:
I've put a few of those cross-refs in too. They get the bold monospace formatting, which is visually helpful, even though they don't link anywhere.